### PR TITLE
[Merged by Bors] - feat: allow users to expand `field_simp` into a `simp` call

### PR DESCRIPTION
**Pull request by alexjbest**
This should allow users to debug what `field_simp` is doing more easily.
And a little bit of miscellaneous cleanup of the field_simp doc.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Util/DischargerAsTactic.lean [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex J. Best
+-/
+import Std.Tactic.Exact
+
+/-!
+## Dischargers for `simp` to tactics
+
+This file defines a wrapper for `Simp.Discharger`s as regular tactics, that allows them to be
+used via the tactic frontend of `simp` via `simp (discharger := my_discharger)`.
```
Comment (semorrison) [inline Mathlib/Util/DischargerAsTactic.lean:12]:
```suggestion
used via the tactic frontend of `simp` via `simp (discharger := wrapSimpDischarger my_discharger)`.
```
might be clearer?

**Comment by semorrison** [comment]
bors merge

**Comment by alexjbest** [comment]
can this be put on the queue again?

**Comment by alexjbest** [comment]
bors retry

**Comment by ADedecker** [comment]
bors retry

**Pull request closed without merging**